frontend/www/js/omegaup/components/arena/RunDetails.vue

```diff
@@ -297,5 +330,34 @@ export default class ArenaRunDetails extends Vue {
     const visible = this.groupVisible[group];
     this.$set(this.groupVisible, group, !visible);
   }
+
+  showDataCase(
+    cases: types.ProblemCasesContents,
+    caseName: string,
+    caseType: string,
+  ): string {
+    return cases[caseName] ? cases[caseName][caseType] : '-';
+  }
+
+  showDiff(cases: types.ProblemCasesContents, caseName: string): string {
+    if (!cases[caseName]) {
+      return '-';
+    }
+    const result = diff(`${cases[caseName].out}`, `${cases[caseName].output}`);
+    let span = '';
+    result.forEach((diff) => {
+      if (diff[0] === -1) {
+        span += `<span class="green">${diff[1]}</span>`;
```

Comment:
todas estas cadenas hay que escaparlas con `ui.escape(...)`.

Reply:
Listo, pero si las escapo, me empiezan a aparecer las etiquetas `html` sin formato. Eso como lo puedo arreglar?

Reply:
puedes arreglarlo escapando únicamente lo que se necesita escapar:

```suggestion
        span += `<span class="green">${ui.escape(diff[1])}</span>`;
```

Reply:
Así si se muestra bien. Gracias